src/main/kotlin/racingcar/Car.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar
+
+class Car(private val name: String) {
+    var distance = ""
+    var moveCount = 0
+    val carName = name
+    fun move(num: Int) {
+        if (num > 3) {
+            moveCount++
+            distance += "-"
+            println("$carName : $distance")
+        } else
+            println("$carName : $distance")
```

Comment:
else 문만 지우면 print가 중복으로 들어가지 않아도 정상동작 가능합니다.
```suggestion
            println("$carName : $distance")
```